doc 3.00

gerhardol opened the pull request:
gitextension/gitextensions#5693

draft for doc update

Replacing screenshots
Remove 2.x functionality descriptions
Remove some irrelevant desriptions
Brief add new functionality
Moved to #90: Minor refresh to coloring, matching updated dashboard

This may be split in several PRs, the documentation has not been updated much since it was created, so there are 5 years of changes.
Contributions are welcome

drewnoakes commented on source/plugins.rst:
```diff
@@ -41,7 +43,7 @@ This list is incomplete.
     .. setting:: Fetch all submodules
 
-      If checked, also perform ``git fetch �all`` recursively on all configured
+      If checked, also perform ``git fetch -all`` recursively on all configured
```
```suggestion
     If checked, also perform ``git fetch --all`` recursively on all configured
```

gerhardol commented:
Progressing slowly, more than half should be done by now at least.
I am touching-up and adding information is in some places, this is something someone need to return to, the documentation has more or less not been updated in 5 years.

When I feel that I am done (or get stuck), I suggest merging directly and ask users to review and enhance. (Open a new issue for visibility and ask for help in gitter.)

RussKie commented:
Agree and support.
Thank you!

On Tue, 11 Dec 2018 at 23:08, Gerhard Olsson <notifications@github.com>
wrote:

> Progressing slowly, more than half should be done by now at least.
> I am touching-up and adding information is in some places, this is
> something someone need to return to, the documentation has more or less not
> been updated in 5 years.
>
> When I feel that I am done (or get stuck), I suggest merging directly and
> ask users to review and enhance. (Open a new issue for visibility and ask
> for help in gitter.)
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/gitextensions/GitExtensionsDoc/pull/88#issuecomment-446180321>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AEMyXgyBda1Qp8O4OvLxcYC9VIGt5ydoks5u36BYgaJpZM4ZHZnN>
> .
>

drewnoakes commented on source/plugins.rst:
```diff
@@ -30,6 +101,96 @@ You should have GitFlow installed to use this plugin.
+
+    .. setting:: Fetch all submodules
+
+      If checked, also perform ``git fetch -all`` recursively on all configured
```
```suggestion
      If checked, also perform ``git fetch --all`` recursively on all configured
```

gerhardol commented:
I have refreshed the documentation, replaced all screenshots, added a few missing pieces of info and touched up a few incorrect pieces. There is still a lot of missing information (not much updates since @martinqt wrote this about five years ago) and I have very likely introduced new errors.

However, I believe that the total is much improved suggest to merge this as is without much review and point the documentation to point to this. I will then open a new PR in the main repo with a request to review the documentation by users. This is much easier for them if a "build" already exists and they have something to base improvements on. I have noticed some issues in GE too when trying to get the screenshots.

I will squash this tomorrow (even if approvals was some time ago, I believe this is OK and the best way forward).

@RussKie @spdr870
To complete the documentation someone with admin privileges need to create the release/3.00 branch and point that to master, then move 'latest' branch too (that will retrigger a a build on readthedocs).
